user_management/docker/docker-compose.yaml

```diff
@@ -21,6 +21,7 @@ services:
 
     user-management:
         networks:
+            - transcendence
```

Comment:
What's the point of adding user_management to the transcendence network? 
Normally, only Nginx should be accessible from the outside.

Reply:
![image](https://github.com/tdameros/42-transcendence/assets/50496792/abf9ee16-333a-43c9-bf86-b53a57ca457a)